extensions/devui/deployment/src/main/java/io/quarkus/devui/deployment/menu/DependenciesProcessor.java

```diff
@@ -91,10 +91,10 @@ void createBuildTimeActions(BuildProducer<BuildTimeActionBuildItem> buildTimeAct
     }
 
     private boolean isEnabled() {
-        var value = ConfigProvider.getConfig().getConfigValue("quarkus.bootstrap.incubating-model-resolver");
+        SmallRyeConfig config = (SmallRyeConfig) ConfigProvider.getConfig();
```

Comment:
Isn't safer to use `unwrap`? 
```suggestion
        SmallRyeConfig config = ConfigProvider.getConfig().unwrap(SmallRyeConfig.class);
```